apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -1250,6 +1278,13 @@ export class RushConfiguration {
     return this._yarnOptions;
   }
 
+  /**
+   * The configuration options used by the current package manager.
```

Comment:
This is now being exposed twice. Should we make a note about that?

Reply:
Couldn't hurt